spec/classes/foreman_proxy__plugin__remote_execution__ssh_spec.rb

```diff
@@ -66,7 +66,7 @@
 
         it { should_not contain_exec('generate_ssh_key') }
         it { should_not contain_file('/root/.ssh') }
-        it { should contain_package(package_name) }
+        it { should_not contain_package(package_name) }
```

Comment:
I'd drop this assertion (and `package_name`). There's practically no chance it should ever show up in the catalog and if it does, it's inherited from something else and that's not a problem.

Reply:
Hmmm, good point. I've dropped it.